docs/spec/normative_examples.md

```diff
@@ -679,12 +690,50 @@ status:
 ```
 
 Even when ready, the new revision does not automatically start serving
-traffic, as the route was pinned to revision `def`.
+production traffic, as the route was pinned to revision `def`. It will, however,
```

Comment:
Do you want to use the term "pinned" here?

Reply:
rm'd "pinned" throughout.